<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitordialog</class>
 <widget class="QDialog" name="monitordialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="ideal_t_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40px &quot;Segoe UI&quot;, sans-serif;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="realTempLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40px &quot;Segoe UI&quot;, sans-serif;</string>
   </property>
   <property name="text">
    <string>waiting arduino</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_monitor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="hc_status">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 30px Arial, sans-serif;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
